docs/installing-solidity.rst

```diff
@@ -329,10 +329,10 @@ The following are dependencies for all builds of Solidity:
 +-----------------------------------+-------------------------------------------------------+
 | `z3`_ (version 4.8.16+, Optional) | For use with SMT checker.                             |
 +-----------------------------------+-------------------------------------------------------+
-| `cvc4`_ (Optional)                | For use with SMT checker.                             |
+| `cvc5`_ (Optional)                | For use with SMT checker.                             |
 +-----------------------------------+-------------------------------------------------------+
 
-.. _cvc4: https://cvc4.cs.stanford.edu/web/
+.. _cvc5: https://cvc5.github.io/downloads.html
```

Comment:
Should this link to the download page or their home page?

Reply:
Let's point to the download page since this is about installing dependencies.